lightning/src/ln/channel.rs

```diff
@@ -676,7 +676,7 @@ impl<Signer: Sign> Channel<Signer> {
 		if feerate_per_kw < lower_limit {
 			return Err(ChannelError::Close(format!("Peer's feerate much too low. Actual: {}. Our expected lower limit: {}", feerate_per_kw, lower_limit)));
 		}
-		let upper_limit = fee_estimator.get_est_sat_per_1000_weight(ConfirmationTarget::HighPriority) as u64  * 2;
+		let upper_limit = fee_estimator.get_est_sat_per_1000_weight(ConfirmationTarget::HighPriority) as u64 * 4;
```

Comment:
We can even give more leeway to our users but hardcode a worst top mempool feerate observed on the last 2 or 4 years ?

Reply:
Hmm, I don't think we want to let it get too crazy. Correct me if I'm wrong, but I *think* the channel funder (who is a miner) can burn their counterparty money if they can set the fee very high, and then cause their counterparty's balance to just be dust?